app/snippets/nrf52833-nosd/snippet.yml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+name: nrf52833-nosd
+boards:
+  # Add any new nRF52833 boards via regular expression:
+  # https://docs.zephyrproject.org/latest/build/snippets/writing.html#by-regular-expression
+  nrfmicro_13_52833:
```

Comment:
I actually think this is generic enough that the overlay you added should be the *default* for this snippet, so folks can use this snippet easily with out of tree boards as well.

Can we also add an nRF52840 version of this while we are at it?

Reply:
Removed the `boards` selector, added `nrf52840-nosd` snippet.

Reply:
@krikun98 This is still apply to just the one board.

Reply:
Sorry, must've accidentally committed the wrong revision. Should be fixed now.